src/handlers/dns/DnsStreamHandler.cpp

```diff
@@ -309,6 +313,30 @@ bool DnsStreamHandler::_filtering(DnsLayer &payload, [[maybe_unused]] PacketDire
     return true;
 }
 
+const std::bitset<64> DnsStreamHandler::_process_dns_groups(const Configurable *metrics_config)
+{
+    // default enabled groups
+    std::bitset<64> groups;
+    groups.set(group::DnsMetrics::Counters);
```

Comment:
we should probably define the list of defaults in the header file - otherwise we will forget to add to the defaults here when we add a new group.